sdk/keyvault/Azure.Security.KeyVault.Administration/src/KeyVaultAdministrationClientOptions.cs

```diff
@@ -11,6 +11,8 @@ namespace Azure.Security.KeyVault.Administration
     /// </summary>
     public class KeyVaultAdministrationClientOptions : ClientOptions
     {
+        internal const string CallerShouldAuditReason = "https://aka.ms/azsdk/callershouldaudit/security-keyvault-administration";
```

Comment:
I am getting access denied for this link. Does it need special permissions?

Reply:
Changes look good to me otherwise.

Reply:
You'll need to request to join.  It's owned by the Security folks, but this is the same pattern we used for things like https://aka.ms/azsdk/callershouldaudit/storage-blobs.